da/test/da_test.go

```diff
@@ -184,7 +189,7 @@ func doTestRetrieve(t *testing.T, dalc da.DataAvailabilityLayerClient) {
 		assert.Len(ret.Blocks, cnt, h)
```

Comment:
We should add an assertion that countAtHeight is always equal to 10. And maybe parameterize 10 as the number of blocks per height.

Reply:
How is it not necessarily ten? Are we not always submitting ten blocks?

Reply:
Yes, we are always submitting 10 blocks at a time but if you look closely, we are keeping track of the number of blocks at each DA height not how many blocks were submitted together. Each time we have a `time.Sleep(time.Duration(rand.Int63() % mockDaBlockTime.Milliseconds()))` which means we sleep for a random duration smaller than the `mockDABlockTime`.  If `resp.DA height` remains the same, for example, all the 100 blocks are submitted at one DA height.  So we cannot assert `countAtHeight` is always equal to 10.

Reply:
I see ty for explaining. The closest explicit assert that we can make is to check if countAtHeight is modulo 10 as one batch is always 10 so one height is a multiple of 10 which might be 0.